.github/workflows/ci.yaml

```diff
@@ -98,6 +98,38 @@ jobs:
       - name: Run make verify-imports
         run: |
           make verify-imports
+  verify-generate:
```

Comment:
I see you have a `verify-all` target, could all of these steps be replaced with just running that?

Reply:
Meh, maybe, maybe not, tbh that only got added and will exit on the first error so wouldn't be ideal. Also i think it it is preferable to see all these as separate checks on the PR:

![image](https://github.com/Kuadrant/dns-operator/assets/327352/de9ef51c-32af-45c4-ac6a-f6e72350004e)

Reply:
You could use `make -k verify-all` to mitigate that. I see a few small advantages of doing it that way (quicker run, no need to update actions if check is added) but it is just a preference thing